lib/ansible/module_utils/gitlab.py

```diff
@@ -59,3 +72,33 @@ def findGroup(gitlab_instance, identifier):
         return None
 
     return project
+
+
+def gitlabAuthentication(module):
+    gitlab_url = module.params['api_url']
+    validate_certs = module.params['validate_certs']
+    gitlab_user = module.params['api_username']
+    gitlab_password = module.params['api_password']
+    gitlab_token = module.params['api_token']
+
+    if not HAS_GITLAB_PACKAGE:
+        module.fail_json(msg=missing_required_lib("python-gitlab"), exception=GITLAB_IMP_ERR)
+
+    try:
+        # python-gitlab library remove support for username/password authentication since 1.13.0
+        # Changelog : https://github.com/python-gitlab/python-gitlab/releases/tag/v1.13.0
```

Comment:
Small nit-pick. Can you please add changelog entry and porting guide note ? Thanks.

Reply:
Just added the changelog.
I this case, there is no need for porting guide because it's only a bugfix